docs/README.md

```diff
@@ -1,20 +1,33 @@
-# box64
-
 ![Official logo](img/Box64Logo.png "Official Logo")
-Linux Userspace x86_64 Emulator with a twist
 
-----
-![box64 build status](https://app.travis-ci.com/ptitSeb/box64.svg?branch=main)
+<p align="center">Linux Userspace x86_64 Emulator with a twist
+<p align="center">
+  <a href="https://github.com/ptitSeb/box64/blob/main/docs/CHANGELOG.md">
+    View changelog</a>
+  |
+  <a href="https://github.com/ptitSeb/box64/blob/main/docs/README_CN.md">
+    中文</a>
+  |
+  <a href="https://github.com/ptitSeb/box64/issues/new">
+    Report an error</a>

```

Comment:
I don't like having html code inside the makdown file. I prefer to keep markdown syntax only.

Reply:
Hmm... is there any way to center text in markdown without using html?

Reply:
markdown doesn't support aligning. github markdown supports with HTML.

Reply:
Yes, but I don't like poluting markdown file with html. Please don't try to align.